app/src/main/java/com/codesoom/assignment/exceptions/TaskAlreadyExistException.java

```diff
@@ -0,0 +1,14 @@
+package com.codesoom.assignment.exceptions;
+
+import org.springframework.http.HttpStatus;
+
+/**
+ * 중복된 할 일을 등록할 때 던집니다.
+ */
+public class TaskAlreadyExistException extends CommonException {
```

Comment:
이 정도가 적절할 것 같습니다. 어쩌면 Task가 이미 존재하는 것은 문제가 아니게 될 지도 모릅니다.
```suggestion
public class TaskDuplicationException extends CommonException {
```

Reply:
(47eecc5) 수정하였습니다.